File: AdotePetJavaProjetct/src/adotepet/fabrica/Adotante.java
package adotepet.fabrica;

public class Adotante extends Pessoa implements ITipoPessoa{
    private char comprovanteResidencia;

    public Adotante(String nome, String cpf, String endereco, String email, String telefone){
        super(nome, cpf, endereco, email, telefone);
    }

    public char getComprovanteResidencia(){
        return comprovanteResidencia;
    }

    public void setComprovanteResidencia(char comprovanteResidencia){
        this.comprovanteResidencia=comprovanteResidencia;
    }
    
    @Override
    public void criaPessoa(){
        System.out.println("Adotante criado");
    }
}

File: AdotePetWeb/backend/api/model/Adotante.php
<?php

/**
 * @author Thaís
 */
class Adotante {
   
    private $nome;
    private $cpf;
    private $dataNascimento;
    private $endereco;
    private $email;
    private $telefone;
    private $comprovanteResidencia;
   
    public function get_nome() {
        return $this->nome;
    }

    public function get_cpf() {
        return $this->cpf;
    }

    public function get_dataNascimento() {
        return $this->dataNascimento;
    }
    
    public function get_endereco() {
        return $this->endereco;
    }
    
    public function get_email() {
        return $this->email;
    }

    public function get_telefone() {
        return $this->telefone;
    }

    public function get_comprovanteResidencia() {
        return $this->comprovanteResidencia;
    }

    public function set_nome($nome) {
        $this->nome = $nome;
        return $this;
    }
    
    public function set_cpf($cpf) {
        $this->cpf = $cpf;
        return $this;
    }

    public function set_dataNascimento($dataNascimento) {
        $this->dataNascimento = $dataNascimento;
        return $this;
    }
    
    public function set_endereco($endereco) {
        $this->endereco = $endereco;
        return $this;
    }
    
    public function set_email($email) {
        $this->email = $email;
        return $this;
    }
    
    public function set_telefone($telefone) {
        $this->telefone = $telefone;
        return $this;
    }

    public function set_comprovanteResidencia($comprovanteResidencia) {
        $this->comprovanteResidencia = $comprovanteResidencia;
        return $this;
    }

    

}
